swap_meet/vendor.py

```diff
@@ -1,2 +1,68 @@
+from swap_meet.item import Item
 class Vendor:
-    pass
\ No newline at end of file
+    
+    def __init__(self, inventory=None):
+        # self.inventory = inventory
+        if inventory is None:
+            inventory = []
+        self.inventory = inventory
+    
+    def add(self, item):
+        self.item = item
+        self.inventory.append(item)
+        return item
+    def remove(self, item):
+        self.item = item
+        if item in self.inventory:
+            self.inventory.remove(item)
+            return item
+        return False
+
+    # create get_by_cateogry instance method that takes in category argument
+
+    def get_by_category(self, category):
+    #     # reutrn list of items by category
+        self.category = category
+        items_in_category = []
+        # instance from item class
+        # item_category = Item(category=self.category)
+        for item in self.inventory:
+            # item_category = Item(category=self.category)
+            # if item.category == item_category.category:
+            if item.category  == category:
+                items_in_category.append(item)
```

Comment:
This would be a great opportunity for a comprehension expression. We could write this as

```py
    items_in_category = [item for item in self.inventory if item.category == category]
```